deno/payloads/v8/application.ts

```diff
@@ -91,6 +91,23 @@ export interface APIApplication {
 	 * See https://discord.com/developers/docs/resources/application#application-object-application-flags
 	 */
 	flags: ApplicationFlags;
+	/**
+	 * Up to 5 tags describing the content and functionality of the application
+	 */
+	tags?: [string, string, string, string, string];
```

Comment:
Only 1 tag is required if specified
```suggestion
	tags?: [string, string?, string?, string?, string?];
```